tool/truffle/jruby_truffle_runner/lib/jruby_truffle_runner.rb

```diff
@@ -0,0 +1,270 @@
+require 'optparse'
+require 'pp'
+require 'yaml'
+require 'fileutils'
+
+# TODO stored after run combinations like -S irb, -I test
+
+class JRubyTruffleRunner
+  attr_reader :options
+
+  begin
+    assign_new_value   = -> (new, old) { new }
+    add_to_array       = -> (new, old) { old << new }
```

Comment:
I guess the GitHub file coloring can't work out the context here.  I thought that was mildly interesting.